client/src/App.js

```diff
@@ -41,10 +44,13 @@ class App extends Component {
             <Route path="/priority-tools" component={PrioritiesTools} />
             <Route path="/error404" component={Error404} />
             <Route path="/errorResources" component={ErrorResources} />
+            <Route path="/yourScenario" component={YourScenario} />
```

Comment:
Hi Haneen! 

I think that the scenario page and persona page should be on one, as the person will need to review both when they are choosing methods / prioritising. Their specific decisions will be based on both scenerio and persona together :)

Reply:
so, you think we need one route "yourScenario" and from it can navigate to personas?